### PR TITLE
test case for web-log analytics app

### PR DESCRIPTION
**Pull request by gsps1**
Test Case for WebLogAnalyticsApplication

### CODE REVIEW

File: src/main/java/co/cask/cdap/guides/WebLogAnalyticsHandler.java [hunk 1]
```diff
@@ -31,6 +30,6 @@ public void getViews(HttpServiceRequest request, HttpServiceResponder responder)
      KeyValue<byte[], byte[]> uri = pageViewScan.next();
      pageViews.put(new String(uri.getKey()), Bytes.toLong(uri.getValue()));
     }
-    responder.sendString(200, pageViews.toString(), Charsets.UTF_8);
+    responder.sendJson(200, pageViews);
```
Comment (abaranau) [inline src/main/java/co/cask/cdap/guides/WebLogAnalyticsHandler.java:33]: will it affect output samples in README? ;)

**Comment by abaranau** [comment]
one comment

**Comment by gsps1** [comment]
updated the README.

**Comment by chtyim** [comment]
LGTM. Merging.

**Pull request merged**